### DISCUSSION TITLE
Apache install documentation?

### DISCUSSION THREAD

**Discussion by bmercernccer** [YOURLS/YOURLS#3501] [General]
I'm trying to get YOURLS working with Apache on a clean Debian 11 instance, but encountering the 404 issue. 

I've done several previous YOURLS installs with Apache and not had these problems, so I think something has changed relatively recently.

Looking at the available documentation, I see it's all oriented towards Nginx, and the troubleshooting suggestions are sparse at best. 

Does anyone have a recent, up-to-date Apache-specific guide to configuring and troubleshooting YOURLS?

**Comment by ozh** [comment]
The whole software and documentation is pretty much "Apache first, the rest may work". See https://yourls.org/docs/guide/install and https://yourls.org/docs/category/troubleshooting

**Reply by bmercernccer** [reply]
I have, of course, already read those pages, but they are extremely sparse on details for Apache configuration.

I'm looking for something with more specific examples of working configurations that I can refer to. Something like the docs in the Awesome YOURLS installation guides, but for Apache instead of Nginx.

**Reply by dgw** [reply]
The very next page after `/install` is https://yourls.org/docs/guide/server-configuration, which has specific examples of Apache config.

**Reply by bmercernccer** [reply]
As I said, I've already read that page. It's not a troubleshooting guide nor is it a how-to.
